scripts/ja/verify-spelling.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##########
+
+# cd to the root path
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd -P)"
+CONTENT_DIR="content/ja"
+cd "${ROOT}/${CONTENT_DIR}"
+
+TARGET_WORDS=""               # invalid:valid
+TARGET_WORDS="${TARGET_WORDS} コンテナー:コンテナ"
+TARGET_WORDS="${TARGET_WORDS} アーキテクチャー:アーキテクチャ"
```

Comment:
We could add more rules, but I'd like to see this is acceptable or not before taking more effort.